### DISCUSSION TITLE
uniswap v3 api is not working

### DISCUSSION THREAD

**Discussion by starman1121** [community/community#36414] [Apps, API and Webhooks] [answered]
I created a swap by forking uniswap v3.
but api is not working now.
please help me.

**Comment by kryptoDev0777** [comment]
hi, nice to meet you again.
uniswap is well-coded protocol.

**Reply by starman1121** [reply]
but api is not working on my server,

**Reply by kryptoDev0777** [reply]
yeah, it will work on your local.
they made API work on uniswap and local.
it will not work on other server

**Comment by kryptoDev0777** [comment]
but all functions will work correctly without API.
they made very well.

**Reply by starman1121** [reply]
yeah, you are right, it works well on local.
but on server, I can't get token price when swapping, 
some prices are coming and somes are not coming

**Reply by kryptoDev0777** [reply]
yeah, maybe.
they cant' get price from uniswap v2 lp.
please check it

**Comment by starman1121** [comment, marked as answer]
you are right, just not coming from uniswap v2.
I think you are expert in this part

**Reply by kryptoDev0777** [reply]
I have ever faced this issue in last month.
hehe.
you should create your own API to fix this problem.
if you want to know more exactly, please dm me @cryptodev0777 (telegram)

**Reply by starman1121** [reply]
Thanks man again
